.github/workflows/test.yaml

```diff
@@ -10,23 +10,27 @@ jobs:
     strategy:
       matrix:
         node-version: [14.x]
-    
+    services:
```

Comment:
There is also a `harmon758/postgresql-action@v1` that would be used the same way as the MySQL one. I wonder which option (a service with a postgres image, or a postgres-action) is faster.

This is an example of how to use it: https://github.com/wizeline/remix-project-lab/blob/main/.github/workflows/deploy.yml#L81

Reply:
> There is also a `harmon758/postgresql-action@v1` that would be used the same way as the MySQL one. I wonder which option (a service with a postgres image, or a postgres-action) is faster.
> 
> This is an example of how to use it: https://github.com/wizeline/remix-project-lab/blob/main/.github/workflows/deploy.yml#L81

Oh nice, didn't know about this action. I ended up using a service in this other PR https://github.com/wizeline/wize-q-remix/pull/259. We are investigating the test failures, but the database connection works. We can look in the future if it's faster to use action.